css/amp.css

```diff
@@ -88,7 +88,8 @@ i-amp-sizer {
 
 .-amp-fill-content {
   display: block;
-  width: 100%;
+  min-width: 100%;
+  width: 1px;
```

Comment:
We don't need `width: 1px`, but let's also add `max-width: 100%` - i checked, that works correctly. Please also add a comment and reference https://bugs.webkit.org/show_bug.cgi?id=155198 to explain why we are doing it.